node/cn/backend.go

```diff
@@ -237,13 +238,14 @@ func New(ctx *node.ServiceContext, config *Config) (*CN, error) {
 
 	config.GasPrice = new(big.Int).SetUint64(chainConfig.UnitPrice)
 
+	mGov := gov_impl.NewGovModule()
```

Comment:
I'm not requesting for change. Just thoughts for the future.

I do think declaring mGov here is the best way so far, but we'll need to rethink how we can inter-link modules and original components.

e.g.
```
Declare modules (NewAbcModule)
Construct components (NewBlockchain, CreateConsensusEngine, ...)
Initialize modules (mAbc.Init)
```